src/pages/marketingPage.js

```diff
@@ -50,9 +49,8 @@ class MarketingPage extends React.Component {
   }
 
   async componentDidMount() {
-    this.props.getLoggedInUser().then(LoggedInUser => {
-      this.setState({ LoggedInUser });
```

Comment:
If component gets updated later (which will happen because `LoggedInUser` is fetched after the mount) then we won't get the updated `LoggedInUser`. We don't need to store the user in the state anymore, we can rely on `this.props` directly.